### ISSUE TITLE
Build a single web file out of everything

### ISSUE THREAD

**Issue by pksunkara** [apiaryio/api-elements.js#21]

**Comment by kylef** [comment]
Note for later, the following behaviour was removed: 

https://github.com/apiaryio/api-elements.js/pull/22/commits/730b7107eb84ea7d913b52c9151df8b6d0af9ce2#diff-698f62fad73a328687b272f66499513cL34

**Comment by opichals** [comment]
Here I would expect to get separate parsers in separate bundles. Like for example:

```
fury.min.js
fury-adapter-apib-parser.min.js
```

**Comment by kylef** [comment]
@opichals Yup that is intended, I think Pavan's wording here was incorrect in the title. Each package should include a browser bundle. Especially for those who don't even deal with parsing but need just the core Minim elements.
